### ISSUE TITLE
gulp watch-bs not watch changes.

### ISSUE THREAD

**Issue by mantrone** [understrap/understrap#960]
Hi,

I manually downloaded understrap from this repo and ran `npm install`. As always I changed the browser-sync options in the file `/gulpconfig.json`, then run `$ gulp watch-bs`.

There are no errors in the terminal, but when I save changes to my files, whether they are `.scss` or `.js`, no compilation or page refresh occurs. This problem does not occur if I download the version of Understrap directly from the Wordpress backend. 

I am not a Node expert and I really need help.
Any ideas of what might be the problem? 

**NPM**: 6.4.1
**Node**: 10.15.3
**Gulp CLI**: 2.0.1

**Comment by holger1411** [comment]
Hi!
(Hopefully) I fixed it here: https://github.com/understrap/understrap/commit/bbae68f0830a9ef8d53e0b7eecd55caebc4d3a09

We run the task as a series after the switch to gulp 4. But that means the watch task for styles and scripts will only be executed AFTER the browser-sync task was done. While the browser-sync task is also an ongoing watcher task, the second watcher did not run in any case...
I set them to "parallel" instead of "series" so both tasks will start at the same time.
